drivers/bus/isapnp/isapnphw.h

```diff
@@ -60,6 +72,8 @@ extern "C" {
 #define ISAPNP_IS_LARGE_TAG(t) (((t) & 0x80))
 #define ISAPNP_LARGE_TAG_NAME(t) (t)
 #define ISAPNP_TAG_MEMRANGE 0x81
+#define    MEMRANGE_16_BIT_MEMORY (0x10 | 0x08)
+#define    MEMRANGE_32_BIT_MEMORY_ONLY 0x18
```

Comment:
Both defines have the same value `0x18`, except the way of notation, is this correct?